src/utils/MultiInviter.ts

```diff
@@ -212,32 +212,29 @@ export default class MultiInviter {
 
                 logger.error(err);
 
-                const isSpace = this.roomId && this.matrixClient.getRoom(this.roomId)?.isSpaceRoom();
+                const room = this.roomId && this.matrixClient.getRoom(this.roomId);
+                const isSpace = room?.isSpaceRoom();
+                const isFederated = room?.currentState.getStateEvents(EventType.RoomCreate, '')
+                    ?.getContent()['m.federate'];
 
+                const type = isSpace ? 'space' : 'room';
                 let errorText: string;
                 let fatal = false;
                 switch (err.errcode) {
                     case "M_FORBIDDEN":
-                        if (isSpace) {
-                            errorText = _t('You do not have permission to invite people to this space.');
+                        if (isFederated === false) {
+                            errorText = _t("This %(type)s is unfederated. " +
+                                "You cannot invite people from external servers.", { type });
                         } else {
-                            errorText = _t('You do not have permission to invite people to this room.');
+                            errorText = _t("You do not have permission to invite people to this %(type)s.", { type });
```

Comment:
This is not i18n'd. It'd say `space` and `room` literally in translations in other languages. Please see https://github.com/vector-im/element-web/blob/develop/docs/translating-dev.md#things-to-knowstyle-guides